[2021-02-23 14:59:25,520] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_passing_params_via_test_command.run_this 2021-02-22T00:20:00+00:00 [queued]>
[2021-02-23 14:59:25,575] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_passing_params_via_test_command.run_this 2021-02-22T00:20:00+00:00 [queued]>
[2021-02-23 14:59:25,577] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:59:25,580] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:59:25,583] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:59:25,840] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): run_this> on 2021-02-22T00:20:00+00:00
[2021-02-23 14:59:26,013] {standard_task_runner.py:52} INFO - Started process 319 to run task
[2021-02-23 14:59:26,273] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_passing_params_via_test_command', 'run_this', '2021-02-22T00:20:00+00:00', '--job-id', '125', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_passing_params_via_test_command.py', '--cfg-path', '/tmp/tmpumk3460c', '--error-file', '/tmp/tmpuqwqwn0j']
[2021-02-23 14:59:26,298] {standard_task_runner.py:77} INFO - Job 125: Subtask run_this
[2021-02-23 14:59:27,530] {logging_mixin.py:104} INFO - Running <TaskInstance: example_passing_params_via_test_command.run_this 2021-02-22T00:20:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:59:28,328] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_passing_params_via_test_command
AIRFLOW_CTX_TASK_ID=run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-02-22T00:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-22T00:20:00+00:00
[2021-02-23 14:59:28,339] {logging_mixin.py:104} INFO -  'miff' was passed in via task params = agg
[2021-02-23 14:59:28,376] {python.py:118} INFO - Done. Returned value was: 1
[2021-02-23 14:59:28,726] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_passing_params_via_test_command, task_id=run_this, execution_date=20210222T002000, start_date=20210223T145925, end_date=20210223T145928
[2021-02-23 14:59:31,607] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-02-23 14:59:31,615] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 319
[2021-02-23 14:59:33,834] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-02-23 14:59:34,020] {process_utils.py:66} INFO - Process psutil.Process(pid=319, status='terminated', exitcode=1, started='14:59:25') (319) terminated with exit code 1
